src/containers/InternalContactOverview.js

```diff
@@ -331,12 +108,12 @@ class InternalContactOverview extends PureComponent {
               }}
             />
             <Divider />
-            {companies.map((company) => (
```

Comment:
would leave it like this since cpy is a bit hard to understand. I guess it's ok right now but for the future it could be confusing ;)

Reply:
I see what you mean... Table name for now is company.. that is why I did not put company as parameter. If we have time we can change company table to companies and make cpy then company. :smile:

Reply:
Table names should be plural. yes. 🥇

Reply:
haha so a interesting read on this subject can be found on https://stackoverflow.com/questions/338156/table-naming-dilemma-singular-vs-plural-names?page=1&tab=votes#tab-top. For purpose of consistency I will amend the company table name to companies. :smile: